src/Native/Common/pal_utilities.h

```diff
@@ -138,3 +139,13 @@ inline static int ToFileDescriptorUnchecked(intptr_t fd)
 {
     return static_cast<int>(fd);
 }
+
+/**
+* Checks if the IO operation was interupted and needs to be retried.
+* Returns true if the operation was interupted; otherwise, false.
+*/
+template <typename TInt>
+static inline bool CheckIo(TInt result)
```

Comment:
I was never particularly happy with my choice of CheckIo as a name.  Now that this function is entirely about EINTR, maybe something like TrueIfInterrupted or CheckInterrupted or something like that would be better.